Add setting unwantedControlPanelsFields and use it in the function filterControlPanelsSchema (#4819)
plone/volto: docs/source/configuration/settings-reference.md

    The group can be one of the default groups 'General', 'Content', 'Security', 'Add-on Configuration', 'Users and Groups' or a custom group.

> filterControlPanelsSchema

Reviewer: Keep this. See next comment for reason.

>     A schema factory for a control panel. It is used internally, to tweak the schemas provided by the controlpanel endpoint, to make them fit for Volto.

Reviewer: I'm not sure what version number this will get released into, so I put the next unreleased version number for now. It might need to be updated when it is actually released.

````suggestion
    ```{deprecated} Volto 16.20.8
    `filterControlPanelsSchema` was renamed to `unwantedControlPanelsFields` in Volto 16.20.8.
    ```
````

>     Control panels fields that are not used on the Volto. It is used internally, to tweak the schemas provided by the controlpanel endpoint, to make them fit for Volto.

Reviewer: This also might need a version number tweak at release.

````suggestion
    Control panels fields that are not used on the Volto.
    It is used internally to tweak the schemas provided by the `controlpanel` endpoint, to make them fit for Volto.

    ```{versionchanged} Volto 16.20.8
    `filterControlPanelsSchema` was renamed to `unwantedControlPanelsFields` in Volto 16.20.8.
    ```
````

    The group can be one of the default groups 'General', 'Content', 'Security', 'Add-on Configuration', 'Users and Groups' or a custom group.

filterControlPanelsSchema

>     A schema factory for a control panel. It is used internally, to tweak the schemas provided by the controlpanel endpoint, to make them fit for Volto. Uses the ``unwantedControlPanelsFields`` setting.

Reviewer: ```suggestion
    A schema factory for a control panel.
    It is used internally, to tweak the schemas provided by the `@controlpanels` endpoint, making them fit for Volto.
    It uses the `unwantedControlPanelsFields` setting.
```

unwantedControlPanelsFields

>     Control panels fields that are not used on the Volto. It is used internally by ``filterControlPanelsSchema`` function.

Reviewer: ```suggestion
    Control panels fields that are not used in Volto.
    It is used internally by the `filterControlPanelsSchema` function.
```

Author: Done.

    The group can be one of the default groups 'General', 'Content', 'Security', 'Add-on Configuration', 'Users and Groups' or a custom group.

filterControlPanelsSchema
    A schema factory for a control panel.
    It is used internally, to tweak the schemas provided by the `@controlpanels` endpoint, making them fit for Volto.
    It uses the `unwantedControlPanelsFields` setting.

unwantedControlPanelsFields

>     Control panels fields that are not used in Volto.

Reviewer: ```suggestion
    Control panel fields that are not used in Volto.
```

Author: The fields can be from several control panels. For example: navigation, site...

That's why it's in the plural. @davisagli  what is your opinion here?

Reviewer: Ah, in that case, it should be something else, using the plural possessive form. I edited my suggestion.
